ui/common/ProgressBar.tsx

```diff
@@ -0,0 +1,30 @@
+'use client'
```

Comment:
I created a progress bar [here](https://codesandbox.io/s/progress-bar-7yheeh?file=/src/App.js) that has gpu animation, lets use that.

Reply:
@afterburn According to the new design for the mining challenge, I think we might have to rework the progress bar. figma mockup - https://www.figma.com/file/LqjK3Tpvd9KJ4buFArCJBQ/Saving-Satoshi?node-id=3427-35637&t=L7QZCyGO0OpLQyT5-0

Reply:
Yeah thats fine, just make sure you use the setup I provided in the codesandbox but feel free to modify it to match the design. Just want to make sure all our animations run on the gpu.